package.json

```diff
@@ -32,6 +32,7 @@
   "author": "Ben Coe <ben@npmjs.com>",
   "license": "ISC",
   "dependencies": {
+    "@c88/v8-coverage": "^0.1.0",
```

Comment:
Could you give a commentary about this package? 🤔

Reply:
This is interesting. I can see a future where c8 becomes a mono repo... 😅